jobs/JOBSPROC_CDAS_DUMP

```diff
@@ -161,7 +161,7 @@ export FIXbufr=${FIXbufr:-$HOMEbufr_dump/fix}
 # TANK is root directory path to observational database used in dumpjb
 #  and engice, sst & snowdepth grib files used in model script
 # --------------------------------------------------------------------
-export DCOMROOT=${DCOMROOT:-/lfs/h1/ops/${envir}/dcom}
+export DCOMROOT=${DCOMROOT:-/lfs/h1/ops/prod/dcom}
```

Comment:
Are NCO stepping away from ${envir}?

Reply:
Yes, in the context of DCOMROOT they are. DCOMROOT does not have a para or test area. Instead it is prod or dev, which are explicitly specified.